src/components/ui/ErrorPage.svelte

```diff
@@ -46,6 +46,10 @@
 				</Button>
 				{@render extraButtons?.()}
 			</div>
+		{:else if extraButtons}
```

Comment:
i think it would be nicer just to move the `{#if href && linkLabel}` one line down, right above the Button element. that should also fix the issue with no code duplication

Reply:
Admittedly half asleep but I had to move the closing {/if} as well to stop svelte screaming at me

~~I swear my experience isn't just Myspace~~